docs/user/README.md

```diff
@@ -18,6 +18,8 @@ To get started adding Glean to your project, choose one of the following guides:
   - Get started adding Glean to an iOS application or library.
 - **[Python](./user/adding-glean-to-your-project/python.md)**
   - Get started adding Glean to any Python project.
+- **[Rust](./user/adding-glean-to-your-project/rust.md)**
+  - Get started adding Glean to any Rust project or library.
```

Comment:
Would you mind removing it from the comment further below that calls it out as a todo?

Reply:
sure thing!